app/views/mailers/digest_mailer/digest_email.html.erb

```diff
@@ -5,17 +5,17 @@
 <ul>
   <% @articles.each do |article| %>
     <li style="margin:18px auto">
-      <a href="https://dev.to<%= article.path %>" style="font-weight: bold;color:#0045ff;font-size:calc(0.93em + 0.2vw);"><%= article.title.strip %></a> <%= truncate(article.description, length: 120) %>
+      <a href="<%= article_url(article) %>" style="font-weight: bold;color:#0045ff;font-size:calc(0.93em + 0.2vw);"><%= article.title.strip %></a> <%= truncate(article.description, length: 120) %>
     </li>
   <% end %>
      <hr />
 </ul>
 
 
 <div>
-  👋 <b>Looking for work?</b> <a href="https://dev.to/listings/jobs">Browse job openings</a> or create an <a href="https://dev.to/listings/forhire">available for hire</a> listing.
+  👋 <b>Looking for work?</b> <a href="<%= app_url("/listings/jobs") %>">Browse job openings</a> or create an <a href="<%= app_url("/listings/forhire") %>">available for hire</a> listing.
   <br />
-  🔍 <b>Hiring developers?</b> Post your position(s) on <a href="https://dev.to/page/post-a-job"><%= community_name %> Listings</a>.
+  🔍 <b>Hiring developers?</b> Post your position(s) on <a href="<%= app_url("/page/post-a-job") %>"><%= community_name %> Listings</a>.
```

Comment:
question that's outside the scope of this: this page is likely DEV only. I couldn't find it in the controller and locally it results in a 404

Reply:
Yes great observation 😊, so whilst making these changes I had actually made a note of some of the urls in the mailers that seem to be created through our admin: 
1. https://dev.to/community-moderation 
2. /page/post-a-job, 
3. /tag-moderation 

I was going to bring it up on the team oss channel and get feedback on the best way to deal with it.